src/psyclone/transformations.py

```diff
@@ -263,11 +265,6 @@ def validate(self, node, options=None):
                 for message in dep_tools.get_all_messages():
                     if "is only written once." in message:
                         continue
-                    if "which indicates a reduction." in message:
-                        continue
-                    # if "is written to, and does not depend" in message:
-                    #    # As it is, this could be a raise condition, but
-                    #    # we could in turn call the HoistTransformation here
                     messages = "\n".join(dep_tools.get_all_messages())
                     raise TransformationError(
```

Comment:
Please document this case in the :raises: section of the docstring.

Reply:
Please also update the copyright date for this file.

Reply:
Done